data/transition-sites/gds_alphagov_alert-preview.yml

```diff
@@ -0,0 +1,11 @@
+---
+site: gds_alphagov_alert-preview
+whitehall_slug: government-digital-service
+homepage: https://alert.preview.publishing.service.gov.uk/
```

Comment:
I think Bouncer handles this: https://github.com/alphagov/bouncer/blob/master/lib/bouncer/outcome/global_type.rb#L7-L9